presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileBasedSecurity.java

```diff
@@ -52,7 +52,7 @@ public void testAdminCanRead()
         queryRunner.execute(admin, "SELECT * FROM orders");
     }
 
-    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = ".*Access Denied: Cannot select from table tpch.orders.*")
+    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = ".*Access Denied: Cannot select from columns \\[clerk, orderkey, orderstatus, custkey, totalprice, comment, orderdate, orderpriority, shippriority\\] in table or view tpch.orders.*")
```

Comment:
We should keep the original message. Listing out all the columns due to table-level access denied is a bad user experience (especially for tables with hundreds of columns)